public/manifest.json

```diff
@@ -19,7 +20,42 @@
     }
   ],
   "start_url": "./",
-  "display": "standalone",
+  "display": "fullscreen",
   "theme_color": "#000000",
-  "background_color": "#222222"
-}
+  "background_color": "#222222",
+  "screenshots": [
+    {
+      "src": "logo192.png",
+      "sizes": "192x192",
+      "type": "image/png"
+    }
+  ],
+
+
+  "shortcuts": [
+    {
+      "name": "About",
+      "url": "./about",
+      "icons": [
+        {
+          "src": "logo512.png",
+          "sizes": "512x512",
+          "type": "image/png",
+          "purpose": "any"
+        }
+      ]
+    },
+    {
+      "name": "Disclaimer",
+      "url": "./disclaimer",
+      "icons": [
+        {
+          "src": "logo512.png",
+          "sizes": "512x512",
+          "type": "image/png",
+          "purpose": "any"
+        }
+      ]
+    }
+  ]
```

Comment:
Very cool! I'm not totally sure how this works, as I didn't notice this doing anything in my Ubuntu 22.04 environment (running GNOME). Is the `shortcuts` property something that's more for Windows or MacOS environments?

Reply:
[https://web.dev/add-manifest/](https://web.dev/add-manifest/)

Reply:
I see now! Thanks for linking the docs. This is a good addition. :)